.github/workflows/ci.yaml

```diff
@@ -125,5 +125,5 @@ jobs:
       - name: Build and publish
         uses: lsst-sqre/build-and-publish-to-pypi@v3
```

Comment:
You'll need to pin to `v1` because that's the last version where we supported token-based publishing. Here's the README for that version: https://github.com/lsst-sqre/build-and-publish-to-pypi/tree/v1.2.0